[2021-12-05 21:16:48,702] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 manual__2021-12-06T02:13:13.036416+00:00 [queued]>
[2021-12-05 21:16:48,707] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 manual__2021-12-06T02:13:13.036416+00:00 [queued]>
[2021-12-05 21:16:48,707] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:16:48,707] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-05 21:16:48,707] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-05 21:16:48,720] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): runme_2> on 2021-12-06 02:13:13.036416+00:00
[2021-12-05 21:16:48,723] {standard_task_runner.py:52} INFO - Started process 4800 to run task
[2021-12-05 21:16:48,730] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2021-12-06T02:13:13.036416+00:00', '--job-id', '10', '--raw', '--subdir', '/Users/ballakeerthi/opt/anaconda3/envs/airflow/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmpq8e0yb18', '--error-file', '/var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T/tmp6q70f7o0']
[2021-12-05 21:16:48,733] {standard_task_runner.py:77} INFO - Job 10: Subtask runme_2
[2021-12-05 21:16:53,765] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2021-12-06T02:13:13.036416+00:00 [running]> on host zipcodes-MacBook-Pro-3.local
[2021-12-05 21:16:58,802] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-06T02:13:13.036416+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-06T02:13:13.036416+00:00
[2021-12-05 21:16:58,803] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b3/j_ybgjv53p33dy62wlktdr4m0000gp/T
[2021-12-05 21:16:58,804] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20211206" && sleep 1']
[2021-12-05 21:16:58,811] {subprocess.py:85} INFO - Output:
[2021-12-05 21:16:58,819] {subprocess.py:89} INFO - example_bash_operator__runme_2__20211206
[2021-12-05 21:16:59,828] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-05 21:16:59,852] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20211206T021313, start_date=20211206T021648, end_date=20211206T021659
[2021-12-05 21:16:59,899] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-05 21:16:59,963] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
